samples/wifi/shutdown/src/main.c

```diff
@@ -217,11 +217,13 @@ int main(void)
 		int ret;
 		struct ethernet_req_params params;
 
-		/* Set a local MAC address with Nordic OUI */
-		ret = net_if_down(iface);
-		if (ret) {
-			LOG_ERR("Cannot bring down iface (%d)", ret);
-			return ret;
+		if (net_if_is_up(iface)) {
+			/* Set a local MAC address with Nordic OUI */
+			ret = net_if_down(iface);
+			if (ret) {
```

Comment:
We cant check for ` if (ret && ret != -EALREADY)` to avoid the no-op case

Reply:
Good point, but this is backported fix, so, don't want to diverge from main fix.